### ISSUE TITLE
Module '"vue3-calendar-composable"' has no exported member 'Dayjs'

### ISSUE THREAD

**Issue by jakhsu** [staaph/vue-calendar-composable#3]
<img width="1280" alt="type import error" src="https://user-images.githubusercontent.com/69961382/234545683-2ccd3453-0509-4c30-bd5a-eff83e14740c.png">

Followed the usage guide and got this error. I think it's because the `index.d.ts` is not exporting `Dayjs` type correctly.

I made this fix in `index.d.ts`
```
import type { Dayjs } from "dayjs";
export { useCalendar, Date, Current, Dayjs};
```

and the error goes away and demo is working now.

**Comment by staaph** [comment]
weird. it is in
https://github.com/staaph/vue-calendar-composable/blob/main/packages/calendar/index.ts

**Comment by staaph** [comment]
Youre on the wrong version
make sure to use 0.1.4

**Comment by jakhsu** [comment]
Yeah, and also would appreciate steps to develop, build, test so people can submit PRs?

**Comment by staaph** [comment]
thats a main pain point. wanted to use the docs as a playground.
For testing i wanted to introduce vitest but im not sure how to mock dayjs + plugins
